[bugfix] syntax error and strange follow-up error

haslersn opened the pull request:
`install.yml` had a syntax error where

```
when: "{{ item.name }} is not defined"
```

was written instead of:

```
when: item.name is not defined
```

This led to the `postgresql.service` not starting on `@mattermost01`. Apparently
the path to `pg_ctl` wasn't specified correctly. (I don't know why the service
was working before.)

The fix for that follow-up issue involved accessing the postgresql version
(`pg_ver.stdout`) because the correct path is version-dependent.

haslersn committed:
[bugfix] syntax error and strang follow-up error

`install.yml` had a syntax error where

```
when: "{{ item.name }} is not defined"
```

was written instead of:

```
when: item.name is not defined
```

This led to the `postgresql.service` not starting on `@mattermost01`. Apparently
the path to `pg_ctl` wasn't specified correctly. (I don't know why the service
was working before.)

The fix for that follow-up issue involved accessing the postgresql version
(`pg_ver.stdout`) because the correct path is version-dependent.

dasJ commented on templates/postgresql.service.j2:
```diff
@@ -5,9 +5,9 @@ After=network.target
 
 [Service]
 Type=forking
-ExecStart={{ postgres_pg_ctl }} -s -D {{ postgres_data_directory }} start -w -t 120
```
Please get rid of the old variable everywhere, e.g. in README.md and tasks/install.yml

haslersn replied:
I added a commit (24954bf7ee35da3c7060b34f58cc92c574afd326) to the source branch which should do it.

haslersn committed:
Get rid of unused vars `postgres_pg_ctl` and `postgres_default_pg_ctl`

They were unused since the prvious commit e003415d742a550fe5d220495eb28e9a370f3a3c.

SlothOfAnarchy commented on tasks/install.yml:
```diff
@@ -76,11 +76,8 @@
-    - name: postgres_pg_ctl
-      value: pg_ctl
-      default: postgres_default_pg_ctl
+      default: postgres_default_initdb
```
Please add the missing newline

haslersn replied:
Done.

haslersn committed:
install.yml: Add missing newline at end of file

dasJ commented:
@SlothOfAnarchy Review and merge please
